src/main/webapp/app/localci/build-queue/build-queue.component.html

```diff
@@ -516,225 +516,211 @@ <h3 id="build-queue-finished-heading" jhiTranslate="artemisApp.buildQueue.finish
+                            <span jhiTranslate="artemisApp.buildQueue.buildJob.status"></span>
+                            <fa-icon [icon]="faSort" />
+                        </th>
+                        <th class="finish-jobs-column">
+                            <span jhiTranslate="artemisApp.buildQueue.buildJob.buildLogs"></span>
+                        </th>
+                        <th jhiSortBy="buildAgentAddress" class="finish-jobs-column-strings">
+                            <span jhiTranslate="artemisApp.buildQueue.buildJob.buildAgent"></span>
+                            <fa-icon [icon]="faSort" />
+                        </th>
+                        <th jhiSortBy="participationId" class="finish-jobs-column">
+                            <span jhiTranslate="artemisApp.buildQueue.buildJob.participationId"></span>
+                            <fa-icon [icon]="faSort" />
+                        </th>
+                        <th jhiSortBy="repositoryName" class="finish-jobs-column-strings">
+                            <span jhiTranslate="artemisApp.buildQueue.buildJob.repositoryName"></span>
+                            <fa-icon [icon]="faSort" />
+                        </th>
+                        <th jhiSortBy="repositoryType" class="finish-jobs-column">
+                            <span jhiTranslate="artemisApp.buildQueue.buildJob.repositoryType"></span>
+                            <fa-icon [icon]="faSort" />
+                        </th>
+                        <th jhiSortBy="commitHash" class="finish-jobs-column-strings">
+                            <span jhiTranslate="artemisApp.buildQueue.buildJob.commitHash"></span>
+                            <fa-icon [icon]="faSort" />
+                        </th>
+                        <th jhiSortBy="courseId" class="finish-jobs-column">
+                            <span jhiTranslate="artemisApp.buildQueue.buildJob.courseId"></span>
+                            <fa-icon [icon]="faSort" />
+                        </th>
+                        <th class="finish-jobs-column">
+                            <span jhiTranslate="artemisApp.buildQueue.buildJob.duration"></span>
+                        </th>
+                        <th jhiSortBy="buildStartDate" class="finish-jobs-column">
+                            <span jhiTranslate="artemisApp.buildQueue.buildJob.start"></span>
+                            <fa-icon [icon]="faSort" />
+                        </th>
+                        <th jhiSortBy="buildCompletionDate" class="finish-jobs-column">
+                            <span jhiTranslate="artemisApp.buildQueue.buildJob.completion"></span>
+                            <fa-icon [icon]="faSort" />
+                        </th>
+                    </tr>
+                </thead>
+                <tbody>
+                    @for (finishedBuildJob of finishedBuildJobs; track trackIdentity($index, finishedBuildJob)) {
```

Comment:
```suggestion
                    @for (finishedBuildJob of finishedBuildJobs; track finishedBuildJob.id ?? -1) {
```
We want to avoid using functions in the html files

Reply:
This didn't work for some reason. I replaced it with just `finishedBuildJob.id`